src/conditions/conditions.component.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  DataTable,
+  DataTableSkeleton,
+  Layer,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+  Pagination,
+} from '@carbon/react';
+import { ErrorState, formatDate, parseDate, usePagination } from '@openmrs/esm-framework';
+import { CardHeader, EmptyState } from '@openmrs/esm-patient-common-lib';
+import React, { useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { usePatientConditions } from './conditions.resource';
+import styles from './conditions.scss';
+
+type PatientConditionsProps = {
+  patientUuid: string;
+  encounterUuid: string;
+};
+
+const PatientConditions: React.FC<PatientConditionsProps> = ({ encounterUuid, patientUuid }) => {
+  const { t } = useTranslation();
+  const { conditions, error, isLoading, mutate } = usePatientConditions(patientUuid);
+  const [pageSize, setPageSize] = useState(3);
+  const pageSizesOptions = useMemo(() => [3, 5, 10, 20, 50, 100], []);
```

Comment:
`useMemo()` seems like overkill here? Why not just move `pageSizeOptions` outside of the component.